### ISSUE TITLE
STR calls in Trio

### ISSUE THREAD

**Issue by 47KW** [Clinical-Genomics/scout#1538]
Good day Sir/Madam
this case shows some strange calls of STR ATXN8OS from the individuals in a trio. Example; 2019-22270-04 has 4 alleles/calls.
https://scout.scilifelab.se/cust002/F0028964-2/str/variants?variant_type=clinical#

Found by Helena Malmgren and she would appriciate to be on the issue as well.

Enjoy Wednesday

**Comment by 47KW** [comment]
We can see that there is two different repeats, but the 2019-22270-02 calls are not there at all.....cant be right?

**Comment by dnil** [comment]
I'm sorry, I dont quite understand the question. 
I can't confirm four calls for 2019-22270? I can see two calls for in the known pathogenic ATXN8OS plus two for the nearby for 2019-22270-04? Each call is a ./1 or 1/., so valid as each is a potential heterozygote. 

I don't know who 2019-22270-02 is - did you mean 2019-05421-02? I agree I would like to have seen a call from that individual. It looks fairly wt on the IGV view, so may just be a no-call, but I can have a look in the ExHu output to make sure!

**Comment by 47KW** [comment]
Yes, It would be helpful to see 2019-05421

Thats essentially what we need, Right Helena?

20 nov. 2019 kl. 09:34 skrev Daniel Nilsson <notifications@github.com<mailto:notifications@github.com>>:


I'm sorry, I dont quite understand the question.
I can't confirm four calls for 2019-22270? I can see two calls for in the known pathogenic ATXN8OS plus two for the nearby for 2019-22270-04? Each call is a ./1 or 1/., so valid as each is a potential heterozygote.

I don't know who 2019-22270-02 is - did you mean 2019-05421-02? I agree I would like to have seen a call from that individual. It looks fairly wt on the IGV view, so may just be a no-call, but I can have a look in the ExHu output to make sure!

**Comment by dnil** [comment]
:+1: Looking into it!

**Comment by dnil** [comment]
The individual in question is indeed normal (20) for the gene in question. But you should have been able to see that. Something appears to go wrong on loading, possibly because there are multiple entries with size 20 after multi-allelic sites decomposition. Lets keep this issue open, and we'll keep you posted!

**Comment by dnil** [comment]
Yes, there is an issue here. When MIP (SVDB) merges individual ExHu runs into a family file, and subsequently MIP splits the multi allelic entries, things go wrong. I'll describe it in some more detail and transfer the issue to MIP. Thanks a whole bunch for noticing! Good job! :+1:
